pkgs/applications/misc/mepo/default.nix

```diff
@@ -54,30 +53,35 @@ in stdenv.mkDerivation rec {
     runHook preInstall
 
     zig build -Drelease-safe=true -Dcpu=baseline --prefix $out install
+    install -d $out/share/man/man1
+    $out/bin/mepo -docman > $out/share/man/man1/mepo.1
```

Comment:
Ideally we would use installManPage here but then we would still need to write the file to somewhere and wouldn't gain much.